master/internal/api_experiment.go

```diff
@@ -206,7 +208,8 @@ func (a *apiServer) GetExperiments(
 		archivedExpr,
 		userFilterExpr,
 		labelFilterExpr,
-		req.Description,
+		fmt.Sprintf("%%%s%%", req.Description),
```

Comment:
If this is for WHERE x LIKE '%...%' in sql i'd probably push that down to the query and just use this to pass the param.